### ISSUE TITLE
[QA] Problems with AppImage update

### ISSUE THREAD

**Issue by gabi18** [owncloud/client#9682]
Seen with ownCloud-3.0.0-daily20220516.7572.AppImage on openSUSE 15.3.

Steps to reproduce:
- set up a local update-server (https://github.com/fmoc/minimal-update-server) using [clientupdater.yml.gz](https://github.com/owncloud/client/files/8699957/clientupdater.yml.gz)
- set env var OCC_UPDATE_URL and start ownCloud-3.0.0-daily20220516.7572.AppImage

Result: The AppImage isn't updated automatically but suggests to download the update manually (the link "here" doesn't work). 

![update-error](https://user-images.githubusercontent.com/75983861/168595329-acc91cc9-7692-4c9f-915e-e966795d8508.png)

The client log shows an error for setting certificate verify locations:
```
05-16 11:48:56:698 [ info sync.httplogger ]:	"902d4a45-64a3-4c23-a9de-ca28f8d3e8f6: Request: GET http://127.0.0.1:5000/?client=TFNCIFZlcnNpb246CW4vYQpEaXN0cmlidXRvciBJRDoJb3BlblNVU0UKRGVzY3JpcHRpb246CW9wZW5TVVNFIExlYXAgMTUuMwpSZWxlYXNlOgkxNS4zCkNvZGVuYW1lOgluL2EK&version=3.0.0.7572&platform=linux-appimage-x86_64&oem=ownCloud&buildArch=x86_64&currentArch=x86_64&versionsuffix=daily20220516 Header: { User-Agent: Mozilla/5.0 (Linux) mirall/3.0.0.7572-daily20220516 (ownCloud, opensuse-leap-5.3.18-150300.59.63-default ClientArchitecture: x86_64 OsArchitecture: x86_64), Accept: */*, X-Request-ID: 902d4a45-64a3-4c23-a9de-ca28f8d3e8f6, Original-Request-ID: 902d4a45-64a3-4c23-a9de-ca28f8d3e8f6, } Data: []"
05-16 11:48:56:724 [ info sync.httplogger ]:	"902d4a45-64a3-4c23-a9de-ca28f8d3e8f6: Response: GET 200 http://127.0.0.1:5000/?client=TFNCIFZlcnNpb246CW4vYQpEaXN0cmlidXRvciBJRDoJb3BlblNVU0UKRGVzY3JpcHRpb246CW9wZW5TVVNFIExlYXAgMTUuMwpSZWxlYXNlOgkxNS4zCkNvZGVuYW1lOgluL2EK&version=3.0.0.7572&platform=linux-appimage-x86_64&oem=ownCloud&buildArch=x86_64&currentArch=x86_64&versionsuffix=daily20220516 Header: { Content-Type: text/xml, Content-Length: 322, Server: Werkzeug/2.0.3 Python/3.6.15, Date: Mon, 16 May 2022 09:48:56 GMT, } Data: [<?xml version=\"1.0\"?>\n<owncloudclient>\n  <version>2000.10.0.6519</version>\n  <versionstring>ownCloud Client 2.10.1 (build 7389)</versionstring>\n  <web></web>\n  <downloadurl>https://download.owncloud.com/desktop/ownCloud/stable/2.10.1.7389/linux-appimage/ownCloud-2.10.1.7389.AppImage.zsync</downloadurl>\n</owncloudclient>\n]"
05-16 11:48:56:769 [ info gui.updater ]:	AppImageUpdate: "zsync2: error setting certificate verify locations:  CAfile: /etc/pki/tls/certs/ca-bundle.crt CApath: /etc/ssl/certs"
05-16 11:48:56:769 [ info gui.updater ]:	AppImageUpdate: "zsync2: Reading and/or parsing .zsync file failed!"
05-16 11:48:56:769 [ critical gui.updater ]:	Update server reported that update is available, but AppImageUpdate disagrees, aborting
```

**Comment by fmoc** [comment]
I can confirm this bug exists in the latest daily. For some reason, the remote TLS certificate cannot be verified, it seems...

**Comment by gabi18** [comment]
Confirmed fixed with ownCloud-3.0.0-daily20220609.7771.AppImage.

The update to newer version is suggested.

![update-appimage](https://user-images.githubusercontent.com/75983861/172821812-4c4a6fda-7faa-4881-8047-b0ef6488b4c5.png)

After click "OK" and restart the new version runs.

**Comment by gabi18** [comment]
TODO: check whether the link 'here' works in case of a manual update
